packages/select/src/FilterMultiSelect/components/ListBoxSection/ListBoxSection.tsx

```diff
@@ -1,24 +1,54 @@
-import React from "react"
+import React, { ReactNode } from "react"
+import { v4 } from "uuid"
 import { MultiSelectItem } from "../../../types"
 import styles from "./ListBoxSection.module.scss"
-
 export interface ListBoxSectionProps {
   items: MultiSelectItem[]
   /**
    * Becomes an aria-label on the section, informing
    * unsighted users
    */
   sectionName: string
+  /** If provided, will override the aria-label for this group */
+  sectionHeader?: ReactNode
   children: (item: MultiSelectItem) => React.ReactNode
 }
 
 export const ListBoxSection = ({
   items,
   sectionName,
   children,
-}: ListBoxSectionProps): JSX.Element => (
-  <div aria-label={sectionName} className={styles.listBoxSection}>
-    {Array.from(items).map(node => node != undefined && children(node))}
-  </div>
-)
+  sectionHeader,
+}: ListBoxSectionProps): JSX.Element => {
+  const listSectionId = sectionHeader ? v4() : undefined
+  return (
+    <li role="presentation">
+      <ul
+        className={styles.listBoxSection}
+        aria-label={sectionName}
+        aria-labelledby={sectionHeader ? listSectionId : undefined}
```

Comment:
This isn't looking quite right to me yet.

In the story you've added, this ul now has an `aria-labelledby` and `aria-describedby` both pointing to the same thing.

If `sectionName` is required here, can't we always just have that as the label, and then if the header exists, link that up as the description? Which would also mean you could remove the visually hidden section name.

Reply:
oh actually that aria-describedby can go, I meant to remove it as it doesn't actually work in this instance

Reply:
I assume because role presentation should remove it from the accessibility tree and while it still works with aria-labelledby, not so much with aria-describedby

Reply:
Ah ok, that sounds alright then.